WcaOnRails/app/controllers/database_controller.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+class DatabaseController < ApplicationController
+  EXPORT_PUBLIC_FOLDER = Rails.root.join('public', 'export')
+
+  RESULTS_EXPORT_FOLDER = EXPORT_PUBLIC_FOLDER.join('results')
+  DEVELOPER_EXPORT_FOLDER = EXPORT_PUBLIC_FOLDER.join('developer')
+
+  RESULTS_SQL = "WCA_export.sql"
+  RESULTS_README = "README.md"
+  RESULTS_METADATA = "metadata.json"
+
+  RESULTS_SQL_PERMALINK = "#{RESULTS_SQL}.zip".freeze
+  RESULTS_TSV_PERMALINK = "WCA_export.tsv.zip"
+
+  DEVELOPER_SQL = "wca-developer-database-dump.sql"
+  DEVELOPER_SQL_PERMALINK = "wca-developer-database-dump.zip"
+
+  RESULTS_README_TEMPLATE = 'database/public_results_readme'
```

Comment:
This line is in single quotes while the lines before it in double quotes. Was it intentional?

Reply:
Nope, Ruby doesn't enforce anything regarding quotes. I wonder why RuboCop didn't complain though...